apps/lib/apps_ui.c

```diff
@@ -136,7 +136,7 @@ void destroy_ui_method(void)
     }
 }
 
-const UI_METHOD *get_ui_method(void)
+UI_METHOD *get_ui_method(void)
```

Comment:
Should we revert this change too?

Reply:
I suggest leaving this in because it may be useful in other situations.